pom.xml

```diff
@@ -620,7 +620,7 @@
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-math3</artifactId>
-                <version>3.2</version>
+                <version>3.5</version>
```

Comment:
Latest version is 3.6.1. Should we use that instead?

Release notes: http://commons.apache.org/proper/commons-math/changes-report.html